packages/core-mobile/app/new/routes/(signedIn)/(modals)/collectibleManagement/index.tsx

```diff
@@ -86,4 +91,39 @@ const CollectibleManagementScreen = (): JSX.Element => {
   )
 }
 
+const CollectibleManagementOptions = () => {
+  const collectibleUnprocessableVisibility = useSelector(
+    selectCollectibleUnprocessableVisibility
+  )
+  const dispatch = useDispatch()
+
+  function handleChange(): void {
+    dispatch(toggleCollectibleUnprocessableVisibility())
+  }
+
```

Comment:
I think we need more padding between "Manage List" and the search bar as well as the search bar and the list content
<img width="352" alt="Screenshot 2025-03-28 at 12 19 49 AM" src="https://github.com/user-attachments/assets/adb9873a-7ceb-4381-ac09-35b190c4d948" />